e2e/src/tests/defaultModelSettings.test.ts

```diff
@@ -68,7 +68,7 @@ test(
         borders.forEach((borderColor) => {
           expect
             .soft(borderColor, ExpectedMessages.defaultTalkToIsValid)
-            .toBe(Colors.highlightedEntity);
+            .toBe(Colors.blue);
```

Comment:
Previous variant with `highlightedEntity` more semantic than `blue` which in future can be not blue
See https://github.com/epam/ai-dial-chat/issues/245
I would suggest to use meaningful/semantic color names instead of specific ones (here and everywhere)